src/bun.js/node/node_fs_watcher.zig

```diff
@@ -364,11 +364,6 @@ pub const FSWatcher = struct {
                     }
                 },
                 .directory => {
-                    // macOS should use FSEvents for directories
```

Comment:
@cirospaciari I removed this and everything seemed to work ok

Reply:
It's OK to remove, maybe add a debug assert, I just added this to see if I did not forget any place, MacOS should use FSEvents for directories.